tests/acceptance/features/apiSpaces/moveSpaces.feature

```diff
@@ -234,3 +236,66 @@ Feature: move (rename) file
+    And user "Brian" has accepted share "/folderA" offered by user "Alice"
+    And user "Brian" has accepted share "/folderB" offered by user "Alice"
+    And user "Brian" has created a folder "/folderA/ONE" in space "Shares Jail"
+    And user "Brian" has created a folder "/folderA/ONE/TWO" in space "Shares Jail"
+    And user "Brian" has stored id of folder "/folderA/ONE" of the space "Shares Jail"
+    When user "Brian" moves folder "/folderA/ONE" from space "Shares Jail" to "/folderB/ONE" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And for user "Brian" the space "Shares Jail" should contain these entries:
+      | /folderA |
+    And for user "Brian" folder "folderB" of the space "Shares Jail" should contain these entries:
+      | /ONE |
+    And for user "Brian" folder "folderA" of the space "Shares Jail" should not contain these entries:
+      | /ONE     |
+      | /ONE/TWO |
+    And user "Brian" folder "/folderB/ONE" of the space "Shares Jail" should have the previously stored id
+
+
+  Scenario: Moving a file out of a shared folder as the sharer
+    Given user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "test data" to "/testshare/testfile.txt"
+    And user "Brian" has created a share with settings
+      | path        | testshare |
+      | shareType   | user      |
+      | permissions | change    |
+      | shareWith   | Alice     |
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Brian" moves file "/testshare/testfile.txt" from space "Personal" to "/testfile.txt" inside space "Personal" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/testfile.txt" for user "Brian" should be "test data"
+    And for user "Alice" folder "testshare" of the space "Shares Jail" should not contain these entries:
+      | testfile.txt |
+    And for user "Brian" the space "Personal" should not contain these entries:
+      | /testshare/testfile.txt |
+
+
+  Scenario Outline: Moving a folder out of a shared folder as the sharer
+    Given user "Brian" has created the following folders
+      | path                     |
+      | /testshare               |
+      | /testshare/testsubfolder |
+    And user "Brian" has uploaded file with content "test data" to "/testshare/testsubfolder/testfile.txt"
+    And user "Brian" has created a share with settings
+      | path        | testshare |
+      | shareType   | user      |
+      | permissions | change    |
+      | shareWith   | Alice     |
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Brian" moves file "/testshare/testsubfolder" from space "Personal" to "/testsubfolder" inside space "Personal" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/testsubfolder/testfile.txt" for user "Brian" should be "test data"
```

Comment:
I think this step does not relate to the scenario.

Reply:
this step is verifying that the moving folder doesn't change the content.